### DISCUSSION TITLE
Duty Cycle unnormal hoch - keine Updates möglich

### DISCUSSION THREAD

**Discussion by Tobusk** [OpenCCU/OpenCCU#2328] [🇩🇪 Fragen&Antworten] [answered]
Hi,
sorry ich bin leider am verzweifeln.
Ich habe alle Einstellungen ausprobiert aber mein Duty Cycle geht immer durch die Decke.
Was mach ich falsch?

Ich hab extra zum testen eine neue RaspberryMatic aufgesetzt.
3.69.7.20230506

Und habe nur einen HmIP-PS-2 angelernt. Egal was ich einstelle der Duty Cycle geht immer auf fast 50 % sodass das Update von 2.22.0 auf 2.22.8 vermutlich nie starten wird. Was mache ich falsch?

	
Zyklische Statusmeldung 
- habe ich aus

Anzahl der auszulassenden Statusmeldungen 
 - 255
 
Anzahl der auszulassenden,
unveränderten Statusmeldungen	
- 255

Gerät dient als Router
- aus

MultiCast-Routing
- aus

Routing aktiv
- aus

Müsste doch so am wenigsten senden oder? 
Ich kappiers nicht. Please help!!!

Wäre generell für Tipps zum Duty Cycle dankbar. Hauptsächlich machen die IP Geräte bei mir echt stress. gut habe halt auch viele: Fußbodenheizung mit WTH, Fensterkontakte, Bewegungsmelder, Lichtdimmer schon recht viel aber es muss ja trotzdem möglich sein das die Updates laufen, oder?

**Comment by jens-maus** [comment]
Was genau ist denn nun bitte das Problem? Das dein DutyCycle zeitweise bei 50% liegt? Nun, das ist erstmal so kein Problem, denn da hast du ja noch bis 100% ziemlich Luft. Aber ja, 50% ist nominell etwas hoch, kommt aber drauf an wieviel Funk-Homematic-Geräte du so insgesamt in deinem Park beheimatest. Selbst habe ich rund 200 Geräte im Park und mein DutyCycle in der Nacht liegt so bei 20% und geht tagsüber wegen Bewegungsmelder (eben allgemeiner Mehraktivität im Haus) bis zu 40-50% hoch und das ohne das ich in den Panikmodus gehe.

Bezogen auf RF-basierte Firmware-Updates kann das mitunter aber natürlich dazu führen das Tage bis Wochen vergehen können bis die Updates durch sind und das auch natürlich bei wartenden Updates der DC immer/dauerhaft an die 50-60% rangeht, was aber per se auch nicht problematisch ist. Muss man halt abwarten und Geduld haben. Oder (wenn man ungeduldig ist) eine zweite Ausweichzentrale mit einem anderen Funkmodul haben die man nur für die Updates hat. Dann das Gerät von der anderen Zentrale ablernen -> an der anderen Anlernen und dort die Updates durchführen lassen und dann alles wieder zurücklernen auf die eigentliche produktive Zentrale. Mühsig, daher würde ich sagen einfach abwarten und Tee trinken auch wenn das Wochen dauern kann und man natürlich während dieser Zeit die Zentrale nicht neustarten sollte weil sonst der Firmware-Update Prozess wieder von vorne beginnt.

Das allgemeine Lebensziel sollte aber natürlich nicht sein dafür zu sorgen das der DC immer gen 0 tendiert, sondern das er in einem akzeptablen Bereich liegt. Und für die normalen Funktionen empfinde ich 50% maximal als Wert der i.O. ist. 

Deine ausgeführten Optimierungen bzgl. Statusmeldungen, Router, Multicasing, usw. sind auch IMHO eher kontraproduktiv im allgemeinen Sinne. Wenn du wirklich den DC minimieren willst musst du ein Auge auf Bewegungsmelder, Energiezähler und ähnliches haben die quasi bei Events einfach öfters etwas senden und natürlich auf deine Programmlogik in der WebUI. D.h. das erste wäre alle Programme mal zu deaktivieren, >1h zu warten ob sich der DC senkt und dann sukzessive diese wieder anzuschalten bis du das findest das am meisten DC generiert und dann dieses versuchst zu optimieren.

**Reply by Tobusk** [reply]
Hallo und danke für deine schnelle Antwort.

Wie oben beschrieben habe ich eine neue Zentrale extra für das Update aufgesetzt und dort nur **EIN** Gerät angelernt und komme damit auf 50 % Auslastung, nochmal **nur mit einem Gerät** - mit dem ebenfalls oben beschriebenen Einstellungen. Aber auch unabhängig von den Einstellungen. Das ist doch nicht normal???

Und "das Lebensziel" sollte sein, das ich nicht jedesmal die Aktoren ablernen muss nur um diese Updaten zu können. Ich habe es so verstanden das wenn der DC nicht unter einen Bereich fällt die Updates gar nicht kommen, oder wie verhält sich das?

**Reply by jens-maus** [reply, marked as answer]
> Wie oben beschrieben habe ich eine neue Zentrale extra für das Update aufgesetzt und dort nur **EIN** Gerät angelernt und komme damit auf 50 % Auslastung, nochmal **nur mit einem Gerät** - mit dem ebenfalls oben beschriebenen Einstellungen. Aber auch unabhängig von den Einstellungen. Das ist doch nicht normal???

Treten diese 50% Auslastung auch auf wenn du KEIN Firmware-Update in der Pipeline hast, d.h. dieses nicht gerade übertragen wird? Denn wenn da ein Firmware-Update gerade in der Übertragung zum Gerät ausgeliefert wird ist es vollkommen normal das du bei 50% landest während der Zeit wo das Firmware-Update übertragen wird, denn ein in Auslieferung befindliches FW-Update erhöht ja logischerweise auch den DutyCycle.

> Und "das Lebensziel" sollte sein, das ich nicht jedesmal die Aktoren ablernen muss nur um diese Updaten zu können. Ich habe es so verstanden das wenn der DC nicht unter einen Bereich fällt die Updates gar nicht kommen, oder wie verhält sich das?

Ja, so ist das. Ich hab die grenze aktuell nicht im Kopf aber meines Wissens liegt die bei rund 50-60%, denn er überträgt das FW-Update eben in kleinen scheiben nur bis zu dieser Grenze damit du für die normalen Operationen noch genug Luft hast. Ich hab selbst so eine extra "Update-Zentrale" z.B. auch nicht und hab alle Updates problemlos in Tagen/Wochen durchbekommen.

**Comment by Baxxy13** [comment]
Eine extra "Update-Zentrale" mit nur einer PS-2 dran dürfte (wenn kein Verbraucher an der PS-2 hängt) so um die 1-3% DC haben. Irgendwelche Fummelleien an den Geräteeinstellungen sind nicht nötig. 
Wenn du das FW-Update auf die Zentrale geladen hast ist ein DC um die 50% normal.

**Reply by Tobusk** [reply]
Ah ok dann verursacht die FW selbst den hohen DC, daran hab ich gar nicht gedacht. 
Das kann natürlich dann sein, man sieht ja nicht wirklich ob er überträgt bzw. einen Fortschritt, das wäre vielleicht eine coole Info. ;-)

**Reply by jens-maus** [reply]
> Ah ok dann verursacht die FW selbst den hohen DC, daran hab ich gar nicht gedacht. Das kann natürlich dann sein, man sieht ja nicht wirklich ob er überträgt bzw. einen Fortschritt, das wäre vielleicht eine coole Info. ;-)

Genau so ist es: Da gesetzlich geregelt trägt natürlich JEDE Funkübertragung inkl. des versenden des FW-Updates zum DutyCycle bei. Und das dieser bei die nur bis 50% geht ist eben dem geschuldet das eQ3 dort in den Routinen eine maximale Grenze eingebaut hat damit die FW-Updates selbst nicht 100% des DutyCycle verbrauchen und damit noch genug "Luft" bleibt. Und dein Wunsch einer Progress-Bar bzw. Info wie weit die Übertragung vorangeschritten ist kannst du gerne an eQ3 herantragen – wir als OpenSource Projekt das keinen Zugriff auf die closed source Firmware-Update Routinen hat können da leider nix dran ändern.

**Reply by Tobusk** [reply]
Danke auf jedenfall !!! 
Ich teste das nochmal aber ich denke das wird es sein. 

Manchmal geht das Update so schnell deshalb wundert man sich warum es dann auf einmal Tage dauert.

**Reply by jens-maus** [reply]
> Manchmal geht das Update so schnell deshalb wundert man sich warum es dann auf einmal Tage dauert.

Sag ja, das Hauptproblem ist: Geduld haben! Ist doch vollkommen wurscht egal ob das Tage oder Wochen braucht. Es gabe bisher ja nie ein Emergency FW-Update das sofort eingespielt werden muss damit das Haus nicht abfackelt. Daher alles nur "Luxusprobleme" IMHO ;-)

**Reply by Tobusk** [reply]
> > Manchmal geht das Update so schnell deshalb wundert man sich warum es dann auf einmal Tage dauert.
> 
> Sag ja, das Hauptproblem ist: Geduld haben! Ist doch vollkommen wurscht egal ob das Tage oder Wochen braucht. Es gabe bisher ja nie ein Emergency FW-Update das sofort eingespielt werden muss damit das Haus nicht abfackelt. Daher alles nur "Luxusprobleme" IMHO ;-)

Ja ich war halt gerade dran Verbindungsprobleme zu analysieren weil bei mir ständig Geräte aussetzen und um die FW auszuschließen habe ich darauf gewartet. und das waren halt 13 Stück oder so. Aber dann weiß ich ja das es irgendwann gehen wird und kann mir den DC erklären und muss da nicht auch noch nach schauen.

**Reply by Tobusk** [reply]
Bricht der eigentlich bei einem Neustart der Zentrale das Update ab und startet neu oder geht's danach einfach weiter und die Daten bleiben erhalten?

**Reply by jens-maus** [reply]
Hatte ich prinzipiell schon gesagt: Kein Neustart zwischendrin machen da nach jeden Neustart der Zentrale der FW-Update Prozess wieder ganz von vorne beginnt.!

**Comment by Baxxy13** [comment]
[Meine Tipps zum Thema...](https://homematic-forum.de/forum/viewtopic.php?f=31&t=69091)

**Reply by Tobusk** [reply]
> [Meine Tipps zum Thema...](https://homematic-forum.de/forum/viewtopic.php?f=31&t=69091)

Yes da stehts, bricht ab. Danke!!!
